include/mbedtls/sha1.h

```diff
@@ -103,7 +105,9 @@ void mbedtls_sha1_free( mbedtls_sha1_context *ctx );
  *                 stronger message digests instead.
  *
  * \param dst      The SHA-1 context to clone to.
+ *                 Must not be \c NULL.
```

Comment:
I'm not sure, how critical it is at this moment, but in other PRs we assumed expressing the requirement for the context pointers to be "This must be initialized", except for the `..._init`, function, which only needs a valid, non-null pointer.

Reply:
Fixed.